cmake/openenclave-config.cmake.in

```diff
@@ -38,6 +50,50 @@ if (UNIX)
       set_target_properties(dl PROPERTIES IMPORTED_LOCATION ${DL_LIB})
     endif ()
   endif ()
+elseif (WIN32)
+  if (NOT NUGET_PACKAGE_PATH)
+    message(FATAL_ERROR "NUGET_PACKAGE_PATH not defined. Please define NUGET_PACKAGE_PATH as the path to the Intel and DCAP Client nuget packages.")
+  endif()
+endif ()
+
+if (NOT TARGET openenclave::sgx_enclave_common)
+  if (UNIX)
+    find_library(SGX_ENCLAVE_COMMON_LIB NAMES sgx_enclave_common)
+  elseif (WIN32)
+    find_library(SGX_ENCLAVE_COMMON_LIB NAMES sgx_enclave_common
```

Comment:
Would it make more sense to use one `find_library` call with `PATHS` conditionally defined?

Reply:
(Or even just use `HINTS` and not care about conditionally defining it.)

Reply:
Thanks Andy, nice suggestion, I like this.